backend/btrixcloud/basecrawls.py

```diff
@@ -688,6 +752,12 @@ async def update_crawl(
     @app.post("/orgs/{oid}/all-crawls/delete", tags=["all-crawls"])
     async def delete_crawls_all_types(
         delete_list: DeleteCrawlList,
+        user: User = Depends(user_dep),
         org: Organization = Depends(org_crawl_dep),
     ):
+        for crawl_id in delete_list.crawl_ids:
+            crawl = await ops.get_crawl_raw(crawl_id, org)
+            if (crawl.get("userid") != user.id) and not org.is_owner(user):
+                raise HTTPException(status_code=403, detail="not_allowed")
```

Comment:
Can make it optional, will only do user check if userid is provided.

Reply:
Passing whole user so we can keep the `org.is_owner(user)` check in there, but otherwise user permissions are moved to `delete_crawls()` now and removed from the various endpoints

Reply:
Done